pkg/devices/holding_register.go

```diff
@@ -11,7 +11,9 @@ import (
 	"github.com/vapor-ware/synse-sdk/sdk/output"
 )
 
+// Handler names that show up in the modbus.yaml.gotmpl.
```

Comment:
This comment should probably be more generalized since non-vapor users may not use the same templating/naming scheme for config files. Perhaps something along the lines of:

> Handler name definitions. These are used in the modbus device config (under the "handler" key) to related devices to a handler.